src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -40,36 +40,17 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.Normalization.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.Normalization.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs">
-      <Link>Common\Interop\Windows\normaliz\Interop.Idna.cs</Link>
-    </Compile>
     <Compile Include="$(CommonPath)\System\StringNormalizationExtensions.Windows.cs">
       <Link>Common\System\StringNormalizationExtensions.Windows.cs</Link>
```

Comment:
when cleaning up the normalization #12168 we can do all necessary change, bu tin general I agree with you.